applications/leaderboard/LeaderboardTest/LeaderboardTest.csproj

```diff
@@ -1,18 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
 
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Google.Cloud.Spanner.Data" Version="2.1.0" />
-    <PackageReference Include="Google.Cloud.Spanner.V1" Version="2.1.0" />
-    <PackageReference Include="JUnitTestLogger" Version="0.6.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
-    <PackageReference Include="xunit" Version="2.2.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
+    <PackageReference Include="JUnitTestLogger" Version="1.1.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
+      <PrivateAssets>all</PrivateAssets>
```

Comment:
Why are the asset specifications needed? If we can get away with just including the reference, I think that would be simpler.
Not to worry if we really need them. Although we could potentially use `PrivateAssets="all" IncludeAssets="all"` as attributes in the element.

We might also want to configure having a common "CommonDependencies.xml" or similar that gets included, so we only need to change one file in future.
These are suggestions for a later PR though :)